docker/MODULE.bazel.docker

```diff
@@ -40,25 +40,14 @@ http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "ht
 # Locally installed dependencies (use in cases of custom dependencies or aarch64)
 ####################################################################################
 
-# NOTE: In the case you are using just the pre-cxx11-abi path or just the cxx11 abi path
-# with your local libtorch, just point deps at the same path to satisfy bazel.
-
-# NOTE: NVIDIA's aarch64 PyTorch (python) wheel file uses the CXX11 ABI unlike PyTorch's standard
-# x86_64 python distribution. If using NVIDIA's version just point to the root of the package
-# for both versions here and do not use --config=pre-cxx11-abi
+# NOTE: In the case you are using just the cxx11 abi path with your local libtorch, just point deps at the same path to satisfy bazel.
```

Comment:
If you are using a local build of torch, just point the Libtorch dep to that path